apps/prairielearn/src/tests/fileEditor.test.ts

```diff
@@ -738,42 +708,39 @@ function doEdits(data) {
     editGet(data.url, false, false, data.contentsA, null);
     // (A, A, A, A)
 
-    badPost('save_and_sync', data.contentsB, data.url);
-    // (A, A, A, A)
-
     editPost('save_and_sync', data.contentsB, data.url, true, false, null);
+    waitForJobSequence(locals, 'Success');
     // (B, B, A, B)
 
-    waitForJobSequence(locals, 'Success');
     pullAndVerifyFileInDev(data.path, data.contentsB);
     // (B, B, B, B)
 
     editGet(data.url, false, false, data.contentsB, null);
     // (B, B, B, B)
 
     writeAndCommitFileInLive(data.path, data.contentsA);
-    // (B, A, B, B)
+    // (A, A, B, B)
```

Comment:
The comment above says that the first element of this tuple is "live at last post"; I think it's fair to say that that would be "A" here?

Reply:
Discussed with Matt; we're going to rename this to "live at last GET" (or: the initial file contents that we would send back with a POST). `writeAndCommitFileInLive` does not perform any requests, so this should indeed be `B`.

Reply:
Updated in 82ca14b.